examples/eu-indicators.py

```diff
@@ -0,0 +1,152 @@
+                ),
+                dbc.Col(
+                    [
+                        html.H2("Select a country"),
+                        dcc.Dropdown(
+                            id="eu-indicators-country-dropdown-x",
+                            options=countries,
+                            value=countries[0],
+                        ),
+                        dcc.Markdown(id="eu-indicators-available-countries-x"),
+                    ]
+                )
+            ]
+        ),
+        dbc.Row(
+            [
+                dbc.Col(
+                    [
+                        html.Div(id="eu-indicators-graph-x"),
+                        html.Div(
+                            [ "Data by ",
+                                html.A("Finnish Institute for Health and Welfare",href=web_url,target="_blank"),
+                            ],
+                            className="text-center fs-3 text",
+                        ),
+                        dbc.Button("Print Page", id="eu-indicators-printing-x", className = 'mt-2'),
+                    ]
+                )
+            ],
+            style={"textAlign": "center"},
+        ),
+        html.Div(id="eu-indicators-printing-hidden-content-x"),
+    ]
+)
+app.clientside_callback(
+    """
+    function(clicks) {
+        if (clicks > 0) {
+          window.print()
+        }
+        return ""
+    }
+    """,
+    Output("eu-indicators-printing-hidden-content-x", "children"),
+    Input("eu-indicators-printing-x", "n_clicks"),
+)
+
+@app.callback(
+    Output("eu-indicators-graph-x", "children"),
+    Input("eu-indicators-indicator-dropdown-x", "value"),
+    Input("eu-indicators-country-dropdown-x", "value")
+)
+def update_indicator_graph(indicator, country):
+
+    dff = df[(df.Indicator == indicator) & (df.Country == country)].dropna(axis=0)
+
+    # In some cases there is only data for one year, so this line prevents notations such as '2016 - 2016'.  
+    title_text = {False: f"{dff.Year.min()} - {dff.Year.max()}", True: str(dff.Year.min())}[
+        dff.Year.min() == dff.Year.max()
+    ]
+
+    traces = []
+    for gender in dff.Gender.unique():
+
+        dfff = dff[dff.Gender == gender]
+
+
+        if len(dfff) > 1:
+            traces.append(
+                go.Scatter(x=dfff.Year.astype(str), y=dfff.Val, name=gender, showlegend=True)
+            )
+        else:
+            traces.append(
+                go.Bar(x=dfff.Year.astype(str), y=dfff.Val,name=gender,showlegend=True)
+            )
+          
+    
+    return dcc.Graph(
+        figure=go.Figure(
+            data=traces,
+            layout=go.Layout(
+                title=dict(
+                    text=f"{indicator} in {country}<br>{title_text} by gender", x=0.5
+                ),
+                xaxis=dict(title="Years"),
+                yaxis=dict(title=indicator),
+                legend = dict(title = 'Gender'),
+                height = 600
+            ),
+        )
+    )

```

Comment:
```suggestion
    # if data has only one year create a bar chart, otherwise a scatter plot
    if dff['Year'].nunique() == 1:
        fig = px.bar(dff, x='Year', y='Val', color='Gender', barmode='group', height=600)
        fig.update_layout(title=dict(text=f"{indicator} in {country}<br>{title_text} by gender", x=0.5),
                          yaxis=dict(title=indicator, tickformat =' '),
                          xaxis=dict(title="Years"))
    else:
        fig = px.line(dff, x='Year', y='Val', color='Gender', height=600)
        fig.update_layout(title=dict(text=f"{indicator} in {country}<br>{title_text} by gender", x=0.5),
                          yaxis=dict(title=indicator, tickformat =' '),
                          xaxis=dict(title="Years"))
                          
return dcc.Graph(figure=fig)

```